src/runtime/test/akms-test.ts

```diff
@@ -34,8 +37,8 @@ describe('arcs key management', function() {
     // Test session key
     const sessionKey = await rewrappedKey.unwrap(cloudKey.privateKey());
     const testBuffer = new Uint8Array([42, 23, 99]);
-    const encryptBuffer = await sessionKey.encrypt(testBuffer, IV);
-    const decryptBuffer = await sessionKey.decrypt(encryptBuffer, IV);
+    const encryptBuffer = await sessionKey['encrypt'](testBuffer, IV);
```

Comment:
Can't we cast this to TestableKey.encrypt?

Reply:
```
src/runtime/test/akms-test.ts:41:32 - error TS2352: Conversion of type 'SessionKey' to type 'TestableKey' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Property 'encrypt' is missing in type 'SessionKey'.

41     const testableSessionKey = sessionKey as TestableKey;
                                  ~~~~~~~~~~~~~~~~~~~~~~~~~
```
:(

Reply:
The methods are there, just that they're hidden, since encrypt/decrypt are @VisibleForTesting only, e.g. they are on WebCryptoSessionKey.  I guess we can stick with quoted properties unless you want to reinterpret_cast<> :)